flow/record/adapter/gcs.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import re
```

Comment:
So we can use some more modern type hinting.

```suggestion
from __future__ import annotations

import logging
import re
```